electron-app/tsconfig.test.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["test"],
+  "exclude": ["node_modules", "src", "dist"]
+}
```

Comment:
since new typescript version that requires to put all source files into root dir, test need to have their own config.